pkg/definitions/task_def.go

```diff
@@ -677,6 +678,72 @@ func (d Definition) GetTask(opts GetTaskOpts) (api.Task, error) {
 	}
 	task.Triggers = triggers
 
+	if d.Permissions != nil {
+		if d.Permissions.RequireExplicitPermissions {
```

Comment:
Nit, I'd switch this `if` case. Seems easier to read.
```suggestion
		if !d.Permissions.RequireExplicitPermissions {
		// If we don't require explicit permissions but permissions have been set. This means that
		// ...
```